### PR TITLE
DevTools - add preliminary StyleSheetActor

### PR DESCRIPTION
**Pull request by codehag**
Based on an error coming from devtools, that the server did not support a stylesheets actor, this patch introduces a [StyleSheetsActor](https://searchfox.org/mozilla-central/rev/26b40a44691e0710838130b614c2f2662bc91eec/devtools/server/actors/stylesheets.js#590-853). It has no methods yet, but it is a small step towards getting the devtools up and running. 

Before the change:
No stylesheet editor

After the change we can see that the style editor now loads. It has no information but I will get to that later.
screen shot 2018-10-14 at 12 51 51

Status after this PR:
* Debugger does not load
* Netmonitor does not load
* Console is loading but doesn't receive messages.
* Inspector is, but does not have any information
* style editor is *now loading* but does not have any information
* Performance is an out of date tab, and will likely need to be changed substantially

next steps for this will be to implement the [protocol methods](https://searchfox.org/mozilla-central/source/devtools/shared/specs/stylesheets.js#74-95)

I am not too sure how to test this, it doesn't look like there are tests yet for the devtools, is that right?

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21941)

### CODE REVIEW

**Comment by jdm** [comment]
Correct; we haven't figured out a way to write meaningful tests for the devtools server.

File: components/devtools/lib.rs [hunk 1]
```diff
@@ -65,6 +66,7 @@ mod actors {
     pub mod performance;
     pub mod profiler;
     pub mod root;
+    pub mod styleSheets;
```
Comment (jdm) [inline components/devtools/lib.rs]: `stylesheets`

File: components/devtools/lib.rs [hunk 2]
```diff
@@ -33,6 +33,7 @@ use actors::network_event::{EventActor, NetworkEventActor, ResponseStartMsg};
 use actors::performance::PerformanceActor;
 use actors::profiler::ProfilerActor;
 use actors::root::RootActor;
+use actors::styleSheets::StyleSheetsActor;
```
Comment (jdm) [inline components/devtools/lib.rs]: `stylesheets`

**Review by jdm — CHANGES_REQUESTED** [review]

**Comment by codehag** [comment]
@jdm if we were to introduce tests, would adding a folder to `tests/unit/` for devtools be a good place to start? that way I could start introducing unit tests.

**Comment by jdm** [comment]
That would be a fine place.

**Pull request merged**